docs/CHANGELOG.txt

```diff
@@ -18,12 +18,50 @@ needs please refer to http://www.prestashop.com for more information.
 
 @author    PrestaShop SA <contact@prestashop.com>
 @copyright 2007-2020 PrestaShop SA and Contributors
+<<<<<<< HEAD
```

Comment:
@matks :warning:

Reply:
AAAaaaaAAAaaaah 😱 my bad !

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/20089/commits/6153dfdd8318ae4c976ba3ff2f51ec22ec4d27a0